### PR TITLE
add cholesky

### PR DESCRIPTION
**Pull request by ForFishes**
add chinese cholesky doc.
https://github.com/PaddlePaddle/Paddle/pull/25250

### CODE REVIEW

**Comment by ForFishes** [comment]
image
image
image

File: doc/fluid/api_cn/tensor_cn/cholesky_cn.rst [hunk 1]
```diff
@@ -1,3 +1,43 @@
+.. _cn_api_tensor_cholesky:
+
 cholesky
 -------------------------------
-**版本升级，文档正在开发中**
+
+.. py:function:: paddle.cholesky(x, upper=False, name=None)
+
+:alias_main: paddle.cholesky
+:alias: paddle.cholesky,paddle.tensor.cholesky,paddle.tensor.linalg.cholesky
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/cholesky_cn.rst]: "," -> ", "

File: doc/fluid/api_cn/tensor_cn/cholesky_cn.rst [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+
+
+
+计算一个对称正定矩阵或一批对称正定矩阵的Cholesky分解。如果 `upper` 是 `True` ，
+则分解形式为 :math:`A = U ^ {T} U` , 返回的矩阵U是上三角矩阵。
+否则，分解形式为 :math:`A = LL ^ {T}` ，并返回矩阵 :math:`L` 是下三角矩阵。
+
+参数：
+    - **x** （Variable）- 输入变量为多维Tensor或LoDTensor，它的维度应该为 `[*, M, N]` ,其中*为零或更大的批次尺寸，并且最里面的两个维度上的矩阵都应为对称的正定矩阵，支持数据类型为float32，float64。
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/cholesky_cn.rst]: 去掉LoDTensor

File: doc/fluid/api_cn/tensor_cn/cholesky_cn.rst [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+返回： 与 `x` 具有相同形状和数据类型的Tensor。它代表了Cholesky分解生成的三角矩阵。
+
+返回类型：  变量（Variable）
+
+**代码示例**
+
+..  code-block:: python
+
+      import paddle
+      import paddle.fluid as fluid
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/cholesky_cn.rst]: no need to import fluid

File: doc/fluid/api_cn/tensor_cn/dot_cn.rst [hunk 4]
```diff
@@ -13,15 +13,16 @@ dot
...
 参数：
-        - **x** （Variable）- 1维 ``Tensor`` 或 ``LoDTensor`` 。数据类型为 ``float32`` 、 ``float64`` 、 ``int32`` 或  ``int64``。
-        - **y** （Variable）- 1维 ``Tensor`` 或 ``LoDTensor`` 。数据类型为 ``float32`` 、 ``float64`` 、 ``int32`` 或  ``int64``。
+        - **x** （Variable）- 1维 ``Tensor`` 。数据类型为 ``float32`` 、 ``float64`` 、 ``int32`` 或  ``int64`` 。
+        - **y** （Variable）- 1维 ``Tensor`` 。数据类型为 ``float32`` 、 ``float64`` 、 ``int32`` 或  ``int64`` 。
         - **name** （str，可选）- 输出的名字。默认值为None。该参数供开发人员打印调试信息时使用，具体用法请参见 :ref:`api_guide_Name` 。
 
 
-返回：  ``Tensor`` 或 ``LoDTensor`` ，数据类型与 ``x`` 相同。
+返回：  ``Tensor`` ，数据类型与 ``x`` 相同。
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/dot_cn.rst:25]: ``Tensor``后多了个空格

File: doc/fluid/api_cn/tensor_cn/dot_cn.rst [hunk 5]
```diff
@@ -33,10 +34,10 @@ dot
     import paddle.fluid as fluid
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/dot_cn.rst]: no need to import fluid

File: doc/fluid/api_cn/tensor_cn/inverse_cn.rst [hunk 6]
```diff
@@ -3,7 +3,7 @@
 inverse
 -------------------------------
 
-.. py:function:: paddle.inverse(input, out=None, name=None)
+.. py:function:: paddle.inverse(x, name=None)
 
 :alias_main: paddle.inverse
 :alias: paddle.inverse,paddle.tensor.inverse,paddle.tensor.math.inverse
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/inverse_cn.rst]: "," -> ", "

File: doc/fluid/api_cn/tensor_cn/inverse_cn.rst [hunk 7]
```diff
@@ -12,29 +12,27 @@ inverse
 
 计算方阵的逆。方阵是行数和列数相等的矩阵。输入可以是一个方阵（2-D张量），或者是批次方阵（维数大于2时）。
 
-**参数**：
-  - **input** (Variable) – 输入张量，最后两维的大小必须相等。如果输入张量的维数大于2，则高维部分代表2-D矩阵的批次（batch）。支持的数据类型：float32，float64。
-  - **out** (Variable，可选) – 指定求和的结果Tensor，可以是程序中已经创建的任何Variable。默认值为None，此时将创建新的Variable来保存输出结果。
+**参数**:
+  - **x** (Variable) – 输入张量，最后两维的大小必须相等。如果输入张量的维数大于2，则高维部分代表2-D矩阵的批次（batch）。支持的数据类型：float32，float64。
```
Comment (jzhang533) [inline doc/fluid/api_cn/tensor_cn/inverse_cn.rst]: 英文部分： When the number of dimensions is greater than 2, it is treated as batches of square matrix. 理解起来比较容易，这里跟英文的保持一致。

File: doc/fluid/api_cn/tensor_cn/cholesky_cn.rst (hunk 1 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/cholesky_cn.rst]: Done

File: doc/fluid/api_cn/tensor_cn/cholesky_cn.rst (hunk 2 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/cholesky_cn.rst]: Done

File: doc/fluid/api_cn/tensor_cn/dot_cn.rst (hunk 4 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/dot_cn.rst:25]: 这个没有空格好像会出问题。

File: doc/fluid/api_cn/tensor_cn/dot_cn.rst (hunk 5 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/dot_cn.rst]: done

File: doc/fluid/api_cn/tensor_cn/inverse_cn.rst (hunk 6 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/inverse_cn.rst]: done

File: doc/fluid/api_cn/tensor_cn/cholesky_cn.rst (hunk 3 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/cholesky_cn.rst]: Done

File: doc/fluid/api_cn/tensor_cn/inverse_cn.rst (hunk 6 above)
Comment (ForFishes) [reply doc/fluid/api_cn/tensor_cn/inverse_cn.rst]: done

**Review by jzhang533 — APPROVED** [review]
lgtm

**Pull request merged**